Sources/SnowManLogicSpecs/ViewSpec.swift

```diff
@@ -12,46 +12,51 @@ class ViewSpec: Swiftest.Spec {
         
        		describe("#receiveGuess") {
         		it("should return the user input if it is a string of letters") {
-            		io.stubbedUserInput = ["hello"]
-                    		expect(view.receiveGuess()).to.equal("hello")
+	            		io.stubbedUserInput = ["hello"]
```

Comment:
Had to modify these tests since I am returning a `Guess` object from the View rather than a `String`